### PR TITLE
mac1 install docs

### PR DESCRIPTION
**Pull request by plaindocs**
I need to flesh this out with specific details.

### CODE REVIEW

**Comment by keithduncan** [comment]
I gave this doc a bit of a rejig while editing it 🙈 

I moved the Elastic CI Stack to the top of the doc so that if you land here looking for "buildkite AWS" it is given precedence before going on to details on how you might run it on your own instances.

I went in to a lot more details on getting the Buildkite Agent to auto-start on macOS. My aim here was to have this section stay the same whether and when we’re able to get a CloudFormation template going for an auto-scaling pool of mac1.metal instances. That is to say, you’ll still have to produce an AMI with `your software + Buildkite Agent` that is plugged in to that CloudFormation stack.

Let me know what you think! :bow:

File: pages/agent/v3/aws.md.erb [hunk 1]
```diff
@@ -1,19 +1,15 @@
 # Running Buildkite Agent on AWS
 
-The Buildkite Agent can be run on AWS using our Elastic CI Stack for AWS.
+The Buildkite Agent can be run on AWS using our Elastic CI Stack for AWS
+CloudFormation template, or by installing the agent on your self-managed
```
Comment (plaindocs) [inline pages/agent/v3/aws.md.erb:4]: I don't know how much we stress, or want to stress, that this is a CF template. @harrietgrace do you have a read on this?

**Review by plaindocs — COMMENTED** [review]
This is excellent. 

> I moved the Elastic CI Stack to the top of the doc so that if you land here looking for "buildkite AWS" it is given precedence before going on to details on how you might run it on your own instances.

:+1: 

> I went in to a lot more details on getting the Buildkite Agent to auto-start on macOS. My aim here was to have this section stay the same whether and when we’re able to get a CloudFormation template going for an auto-scaling pool of mac1.metal instances. 

Perfect. 

I've kicked the broken link and the spellchecker, and done some minor adjustments here there. 

Looks like whatever we use to generate slugs from titles does '`.`' -> `dot` which is pretty weird, and might be worth fixing at some point.

**Comment by plaindocs** [comment]
@keithduncan GH thinks it is my PR, so on you to approve. 
:shipit: as far as I'm concerned.

**Review by keithduncan — APPROVED** [review]

**Comment by keithduncan** [comment]
Thank you for the tweaks, it’s looking great :bow:

**Pull request merged**